man/systemd-cryptenroll.xml

```diff
@@ -394,7 +416,9 @@
         <option>--tpm2-public-key-pcrs=</option>: the former binds decryption to the current, specific PCR
         values; the latter binds decryption to any set of PCR values for which a signature by the specified
         public key can be provided. The latter is hence more useful in scenarios where software updates shell
-        be possible without losing access to all previously encrypted LUKS2 volumes.</para>
+        be possible without losing access to all previously encrypted LUKS2 volumes.
+        Like the <option>--tpm2-pcrs=</option>, string aliases as defined in the table above can also be used
+        to specify the registers. for instance <option>--tpm2-public-key-pcrs=boot-loader+system-identity</option></para>
```

Comment:
Small nit:

```suggestion
        to specify the registers, for instance <option>--tpm2-public-key-pcrs=boot-loader+system-identity</option></para>
```

Reply:
> Small nit:

Good evening sir.
Please I'm not clear on what "Small nit" means. I can only tell you want me to make some changes. Do you mind elaborating a little more? Thank you

Reply:
Oh... I see it was a punctuation you helped correct. '.' , changed to the appropriate ','

Thank you so much sir.

Reply:
> > Small nit:
> 
> Good evening sir. Please I'm not clear on what "Small nit" means. I can only tell you want me to make some changes. Do you mind elaborating a little more? Thank you

The period is not correct here - either it should be a comma or the next word needs to be capitalized.

Reply:
I see it's a suggested changed.

Reply:
> Small nit:

Fixed sir.